spec/models/session_spec.rb

```diff
@@ -32,6 +32,17 @@
         expect(subject.ttl).to be_an(Integer)
         expect(subject.ttl).to be_between(0, 3600)
       end
+
+      it 'will extend a session within the maximum ttl' do
+        subject.created_at = subject.created_at - (described_class::MAX_SESSION_LIFETIME - 1.minute)
+        expect(subject.expire(3600)).to eq(true)
+      end
+
+      it 'will not extend a session beyond the maximum ttl' do
+        subject.created_at = subject.created_at - (described_class::MAX_SESSION_LIFETIME + 1.minute)
```

Comment:
👍 agree that it's simpler, although if the test incrementally pushes out the session is that closer to what actually happens? but again minor nit so cool either way